docs/terminal/cairns.md

```diff
@@ -8,175 +8,182 @@
 
 | Name               | ID      | Callsign       | Frequency        | Login Identifier              |
 | ------------------ | --------------| -------------- | ---------------- | --------------------------------------|
-| **Melbourne Approach East**    |**MAE**| **Melbourne Approach**   | **132.000**         | **ML_APP**                                   |
-| Melbourne Departures North†    |MDN| Melbourne Departures  | 118.900         | ML_DEP          |
-| Melbourne Departures South†   |MDS| Melbourne Departures | 129.400          | ML-S_DEP         |
-| Melbourne Flow†        |MFL|                |          | ML-FLW_CTR                               |
+| **Cairns Approach**    |**CS1**| **Cairns Approach**   | **118.400**         | **CS_APP**          |
```

Comment:
Cairns Approach 1 is demonstrated at CSA in vatSys
Like wise Cairns Approach 2 is CSD

Reply:
Images will have to be updated too